app/containers/EmojiPicker/categories.js

```diff
@@ -1,54 +1,56 @@
-const list = ['frequently', 'custom', 'people', 'nature', 'food', 'activity', 'travel', 'objects', 'symbols', 'flags'];
+import I18n from '../../i18n';
+
+const list = ['frequentlyUsed', 'custom', 'people', 'nature', 'food', 'activity', 'travel', 'objects', 'symbols', 'flags'];
 const tabs = [
 	{
 		tabLabel: '🕒',
 		category: list[0],
-		name: 'Frequently Used'
+		name: I18n.t('Frequently_Used')
```

Comment:
`tabs` constant is created when the bundle is instantiated.
So it's never going to follow I18n changes.
You can do the test:
- Open emoji picker
- Check the label
- Change language
- Check again (it'll never change)

I know you added on `en` only, but I'm pretty sure that's what's going to happen :)

Reply:
Oh damn, I'll sure do the test and if required revert the those changes back 😊

Reply:
Yup, moved the i18n translations to the EmojiCategory fixes this issue :+1: